### PR TITLE
add menu list and separator

### PR DESCRIPTION
**Pull request by nataliadelmar**
Added menu 2 groups of component tokens: 1) menu list and separator
image
image

### CODE REVIEW

File: components/menu-list.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "menu-list":{
```
Comment (paulwitty) [inline components/menu-list.json:2]: Can you change this to "menulist" please?

File: components/label.json [hunk 2]
```diff
@@ -1,5 +1,7 @@
 { 
   "label": {
+    "comment":"Applied to input labels",
```
Comment (paulwitty) [inline components/label.json:3]: This is not correct, it's for generic text which appears on screen.

File: components/code-input.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "codeinput": {
-    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=15403%3A28176",
+    "comment":"Applied to code or 'pin' input",
+    "figma":"https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4931%3A2083",
```
Comment (paulwitty) [inline components/code-input.json:4]: Wrong link

File: components/avatar.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "avatar": {
-    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4594%3A8792",
+    "comment":"applied to avatars and all its elements",
+    "figma":"https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4389%3A1647",
```
Comment (paulwitty) [inline components/avatar.json:4]: These are different links, but appear to go to the same place. Figma is confusing

File: components/label.json (hunk 2 above)
Comment (Coread) [reply components/label.json:3]: I didn't realise that when you made the PR. Does this feed into the localisation pipeline somehow?

Comment (paulwitty) [reply components/label.json:3]: The desktop team wanted a generic component just for showing text on the screen when it isn't part of any other component, and suggested it take the name "label". So unrelated to anything in the provided link.

Also unrelated to anything localisation.

Comment (Coread) [reply components/label.json:3]: Ok, so label is not a keyword but comment is.

Comment (paulwitty) [reply components/label.json:3]: Yeah, "comment" and "figma" are special properties which appear in the input files but are ignored when processing.

File: components/avatar.json (hunk 4 above)
Comment (nataliadelmar) [reply components/avatar.json:4]: the position of the view is slightly different; that is why the link is different. They both take you to the correct component, so Ill mark it as resolved

**Pull request merged**